src/SecTester.Scan/Target/TargetOptions.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using SecTester.Scan.Models;
+
+namespace SecTester.Scan.Target;
+
+public interface TargetOptions

```

Comment:
You should provide the ability to pass a custom query serializer (e.g. `Func<IDictionary<string, IEnumerable<string>>, string> SerializeQuery`)